aikau/src/test/resources/config/Suites.js

```diff
@@ -252,6 +252,7 @@ define(function() {
       "alfresco/renderers/PublishPayloadMixinOnActionsTest",
       "alfresco/renderers/ReorderTest",
       "alfresco/renderers/SearchResultPropertyLinkTest",
+      "src/test/resources/alfresco/renderers/SelectorTest",
```

Comment:
I didn't explicitly mention it (apologies) but I discovered that we don't need the src/test prefix, and so it was removed from all tests.